src/test/java/pl/nextleveldev/smart_route/route/api/RouteControllerTest.java

```diff
@@ -0,0 +1,78 @@
+package pl.nextleveldev.smart_route.route.api;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import pl.nextleveldev.smart_route.route.RouteService;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+class RouteControllerTest {
+
+    public static final String START_LOCATION = "Marszałkowska 10";
+    public static final String DESTINATION = "Wałbrzyska 7";
```

Comment:
Polish diacritics (e.g. `ł`, `ś`, `ż`) can be problematic in certain situations:
- International teams (not every developer uses an IDE with UTF-8 support).
- Encoding issues in CI/CD systems (e.g. old versions of Jenkins, Maven) may have default encoding `ISO-8859-1` - this is a real-life case).
- Issues with external APIs (without using *Percent-encoding* or *URL encoding*).

Suggestion:
```suggestion
public static final String START_LOCATION = "Centrum 01";
public static final String DESTINATION = "KINO FEMINA 01";

// or

// File encoding: UTF-8
public static final String START_LOCATION = "Marszałkowska 01";
public static final String DESTINATION = "KINO FEMINA 01";
```

p.s. Why is the number format in this way `01`? - Right now you have to trust me for a word.